### ISSUE TITLE
Fullcalendar-scheduler.d.ts doesn't export Options

### ISSUE THREAD

**Issue by Brinkzter** [fullcalendar/fullcalendar-scheduler#421]
Hello!

I've been trying to use the fullcalendar-scheduler typings file, but it doesn't seem to work. 

I'm using the scheduler in an Angular component and I want to send the options via an @Input(). But when I want to add OptionsInputBase interface it doesn't recognise the scheduler properties (e.g. resourceAreaWidth). 

I've also tried to use: 'ResourceSourceInput.' But the fullcalendar-scheduler.d.ts doesn't seem to export anything.

```
declare module 'fullcalendar-scheduler/src/main' {
	import 'fullcalendar-scheduler/src/exports'; // <--- doesn't exist
	import 'fullcalendar-scheduler/src/Calendar';
	import 'fullcalendar-scheduler/src/Constraints';
	import 'fullcalendar-scheduler/src/View';
	import 'fullcalendar-scheduler/src/component/DateComponent';
	import 'fullcalendar-scheduler/src/component/InteractiveDateComponent';
	import 'fullcalendar-scheduler/src/component/renderers/EventRenderer';
	import 'fullcalendar-scheduler/src/component/interactions/DateSelecting';
	import 'fullcalendar-scheduler/src/component/interactions/EventDragging';
	import 'fullcalendar-scheduler/src/component/interactions/EventResizing';
	import 'fullcalendar-scheduler/src/component/interactions/ExternalDropping';
	import 'fullcalendar-scheduler/src/models/EventSource';
	import 'fullcalendar-scheduler/src/models/EventDef';
	import 'fullcalendar-scheduler/src/models/EventDefMutation';
	import 'fullcalendar-scheduler/src/timeline/config';
	import 'fullcalendar-scheduler/src/resource-timeline/config';
	import 'fullcalendar-scheduler/src/resource-basic/config';
	import 'fullcalendar-scheduler/src/resource-agenda/config';
	import 'fullcalendar-scheduler/src/types/input-types';
	import 'fullcalendar-scheduler/src/types/jquery-hooks';

}
```

I've added an example of what I've tried on JSBin:
http://jsbin.com/cavoxojiwe/1/edit?js,output

Could anyone help me with this?

Thank you very much!

Henry

**Comment by acerix** [comment]
Thanks for the report.  There is an open issue to add the typings for fullcalendar.d.ts, so this seems the same but for fullcalendar-scheduler.d.ts:

https://github.com/fullcalendar/fullcalendar/issues/4006

I don't see the issue in your JSBin; that seems to be not working correctly because of duplicate IDs in the resources.

**Comment by arshaw** [comment]
the code that i believe will fix this is queued for release:
https://github.com/fullcalendar/fullcalendar-scheduler/commit/44ca37d0d06c67abee5abe07718ffefd8319fb4b
https://github.com/fullcalendar/fullcalendar/issues/4006

**Comment by arshaw** [comment]
fixed in [v1.9.3](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.9.3). could you verify @Brinkzter

**Comment by Brinkzter** [comment]
Hi,

Sorry voor the late reply. OptionsInput is exported in fullcalendar.d.ts, thank you.

But it still keeps complaining that: 'SchedulerLicenseKey' and other scheduler specific properties aren't recognised.
We are trying to set up an Angular component with a @Input() options: OptionsInput. So if you want to use the component with other options, you can do so.

I see that fullcalendar-scheduler/src/exports doesn't exist:
`declare module 'fullcalendar-scheduler/src/main' {
	import 'fullcalendar-scheduler/src/exports'; <-- Doesn't exist
	import 'fullcalendar-scheduler/src/Calendar';
	import 'fullcalendar-scheduler/src/Constraints';
	import 'fullcalendar-scheduler/src/View';
	import 'fullcalendar-scheduler/src/component/DateComponent';
	import 'fullcalendar-scheduler/src/component/InteractiveDateComponent';
	import 'fullcalendar-scheduler/src/component/renderers/EventRenderer';
	import 'fullcalendar-scheduler/src/component/interactions/DateSelecting';
	import 'fullcalendar-scheduler/src/component/interactions/EventDragging';
	import 'fullcalendar-scheduler/src/component/interactions/EventResizing';
	import 'fullcalendar-scheduler/src/component/interactions/ExternalDropping';
	import 'fullcalendar-scheduler/src/models/EventSource';
	import 'fullcalendar-scheduler/src/models/EventDef';
	import 'fullcalendar-scheduler/src/models/EventDefMutation';
	import 'fullcalendar-scheduler/src/timeline/config';
	import 'fullcalendar-scheduler/src/resource-timeline/config';
	import 'fullcalendar-scheduler/src/resource-basic/config';
	import 'fullcalendar-scheduler/src/resource-agenda/config';
	import 'fullcalendar-scheduler/src/types/input-types';
	import 'fullcalendar-scheduler/src/types/jquery-hooks';

}`

Am I using this the wrong way perhaps?

Thank you for your help.
